src/components/search/findly-sdk.js

```diff
@@ -20717,10 +20750,10 @@ FindlySDK.prototype.searchHistroy = function (findlyConfig) {
             messageData.isFromHistory = true;
             //messageData.timestamp = history.timestamp;
           } else {
-            messageData.text = 'Here are the relevant results';
+            messageData.text = langTranslator("sa_sdk_sure_please_find_the_matched_results_below");
```

Comment:
@BharadwajAd-Kore : Please replace this sa_sdk_sure_please_find_the_matched_results_below with 'Here are the relevant results' in i18n sdk key
@SunilSi-kore